rdr_service/alembic/versions/15e05f673f5f_altering_biobank_dv_order_history.py

```diff
@@ -0,0 +1,116 @@
+"""Altering biobank_dv_order_history
+
+Revision ID: 15e05f673f5f
+Revises: f3fdb9d05ab3
+Create Date: 2019-10-04 14:31:57.374442
+
+"""
+from alembic import op
+from sqlalchemy.dialects import mysql
+
+# revision identifiers, used by Alembic.
+revision = '15e05f673f5f'
+down_revision = 'f3fdb9d05ab3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+def upgrade_rdr():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.alter_column('bigquery_sync', 'project_id',
+               existing_type=mysql.VARCHAR(length=80),
+               nullable=True)
+    # ### end Alembic commands ###
+
+    # Procedure to remove the biobank_reference column and move data to correct columns of the history table
+    # Make a bkup_ table
+    op.execute(
+        "CREATE TABLE bkup_biobank_dv_order_history AS SELECT * FROM biobank_dv_order_history;"
+    )
+
+    # PREP THE tmp_ TABLE
+    # create the tmp_ table
+    op.execute(
+        "CREATE TABLE tmp_biobank_dv_order_history AS select * from biobank_dv_order_history;"
```

Comment:
You could add `limit 1` to this statement since you are truncating the new temp table in the next step.